internal/cloud/realm/export.go

```diff
@@ -0,0 +1,72 @@
+package realm
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"mime"
+	"net/http"
+
+	"github.com/10gen/realm-cli/internal/utils/api"
+)
+
+const (
+	exportPathPattern = appPathPattern + "/export"
+
+	exportQueryForSourceControl = "source_control"
+	exportQueryIsTemplated      = "template"
+	exportQueryVersion          = "version"
+
+	mediaParamFilename = "filename"
+
+	trueVal = "true"
+)
+
+// ExportRequest is a Realm application export request
+type ExportRequest struct {
+	ConfigVersion    AppConfigVersion
+	ForSourceControl bool
+	IsTemplated      bool
+}
+
+func (c *client) Export(groupID, appID string, req ExportRequest) (string, io.ReadCloser, error) {
+	options := api.RequestOptions{
+		UseAuth: true,
+		Query: map[string]string{
+			exportQueryVersion: DefaultAppConfigVersion.String(),
+		},
+	}
+
+	if req.ConfigVersion != AppConfigVersionNil {
+		options.Query[exportQueryVersion] = req.ConfigVersion.String()
+	}
+	if req.ForSourceControl {
```

Comment:
do we actually need/want this option? considering that we shouldn't be exporting resource `ObjectId`s anymore, I would assume we would just always want to use this option?

Reply:
yeah, that's a good point.  I'll remove it for now and hard-code us to use this option until we need it to be otherwise